tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/PulsarContainer.java

```diff
@@ -81,10 +82,11 @@ public void start() {
             this.waitStrategy = new HostPortWaitStrategy()
                 .withStartupTimeout(Duration.of(300, SECONDS));
         }
+        this.withLogConsumer(new Slf4jLogConsumer(log).withPrefix(hostname));
```

Comment:
an Slf4jLogConsumer might be helpful if it is a standalone container. but it doesn't help when running testcontainers in a cluster mode. it actually make things a bit worse. so I would prefer put this under a flag (either system environment or system property). e.g `enableSlf4jLogConsumer`.

so people can enable it when running locally, and we can disable it on CI. on CI we should use what @ivankelly has implemented on dumping container logs on tests completed (I am porting that behavior to testcontainers at #2076 )